app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
```

Comment:
Didn't understand why you used range(len())... You can iterate directly over the list to use it's values. This is a better approach:
```suggestion
    for pointer in argv:
```
And if you need the index as well, you can use `enumerate()`:
```suggestion
    for index, pointer in enumerate(argv):
```